src/form.svelte

```diff
@@ -12,13 +12,36 @@
   let saveState = "";
   let errorMessage = "";
   let availableTagNames = []
+  let formTitle = "";
 
   async function init() {
     const tabInfo = await getCurrentTabInfo();
     url = tabInfo.url;
     titlePlaceholder = tabInfo.title;
     const availableTags = await getTags().catch(() => [])
     availableTagNames = availableTags.map(tag => tag.name)
+
+   initCurrentBookmarkData();
+  }
+
+  async function initCurrentBookmarkData() {
+    formTitle = 'Searching for existing Bookmarks';
+
+     var bookmarkSearch = await lookForCurrentBookmark(url)
+        .catch(() => []);
+
+    if ( bookmarkSearch && bookmarkSearch.length > 0 ) {
+        var temp = bookmarkSearch[0];
+        formTitle = 'Edit Bookmark';
+        title = temp.title;
+        tags = (temp.tag_names ? temp.tag_names.join(' ') : "");
```

Comment:
```suggestion
        tags = temp.tag_names ? temp.tag_names.join(" ") : "";
```
Parenthesis are not necessary here. Remaining code uses double quotes, so let's align this here.